content/kb/troubleshooting-rwx-volume-fails-to-attached-caused-by-protocol-not-supported.md

```diff
@@ -24,19 +24,21 @@ The issue applies to RWX volumes on hosts running operating systems that use spe
 
 ## Reason
 
-Longhorn RWX volumes depend on NFS when connecting multiple pods to a shared volume. However, commits to the Linux kernel can occasionally break NFS functionality. A regression of the NFS protocol happened in kernel version 5.15.0-94, and another in version 6.5.6.  Because of the regression, NFS clients are unable to connect to an NFS server inside a share manager pod, and then the attachment operation fails.
+Longhorn RWX volumes depend on NFS when connecting multiple pods to a shared volume. However, commits to the Linux kernel can occasionally break NFS functionality. A regression in the NFS protocol is identified in this [kernel commit](https://github.com/torvalds/linux/commit/51d674a5e4889f1c8e223ac131cf218e1631e423). Because of the regression, NFS clients are unable to connect to an NFS server inside a share manager pod, and then the attachment operation fails.
 
 #### Solution
 
-To address the issue, you can perform either of the following actions:
+The regression has been resolved in the [kernel commit](https://github.com/torvalds/linux/commit/379e4adfddd6a2f95a4f2029b8ddcbacf92b21f9). To resolve the issue, you can examine the source code of your Linux kernel to determine if the error originates from the [commit](https://github.com/torvalds/linux/commit/51d674a5e4889f1c8e223ac131cf218e1631e423). If confirmed, you can then proceed with either of the following actions:
 
 - Upgrade the operating system to a version that uses a fixed kernel.
 - Downgrade the operating system to a version that uses a kernel released before the regression occurred.
 
-| Broken Version | Downgrade Version | Upgrade Version |
-| -------------- | ----------------- | --------------- |
-| 5.15.0-94      | 5.15.0-92         | 5.15.0-100      |
-| 6.5.6          | 6.5.5             | 6.5.7           |
+| Distro         | Broken Version | Downgrade Version | Upgrade Version |
+| -------------- | -------------- | ----------------- | --------------- |
+| Vanilla kernel | 6.5.6          | 6.5.5             | 6.5.7           |
+| Ubuntu         | [5.15.0-94](https://git.launchpad.net/~ubuntu-kernel/ubuntu/+source/linux/+git/jammy/log/?h=Ubuntu-5.15.0-94.104)      | 5.15.0-92         | 5.15.0-100      |
+| Ubuntu         | [6.5.0-21](https://git.launchpad.net/~ubuntu-kernel/ubuntu/+source/linux/+git/jammy/tag/?h=Ubuntu-hwe-6.5-6.5.0-21.21_22.04.1)       |                   |                 |
```

Comment:
For `6.5.0-21`, there is neither a downgrade nor an upgrade version. Does that mean no way to work around it?

Reply:
Actually feel `downgrade version` is a bit weird....

Reply:
I see, but why do we mention downgrade version for other distros here?

Reply:
I think the downgrade versions are from conversations with users.
How about removing the upgrade versions?

Reply:
Probably keep the below good enough instead of specific versions?

>- Upgrade the operating system to a version that uses a fixed kernel.
- Downgrade the operating system to a version that uses a kernel released before the regression occurred.

Reply:
Agree with the idea.

Reply:
Updated. Thank you.